src/vs/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -64,8 +66,7 @@ export class WelcomePageContribution implements IWorkbenchContribution {
 		@IWorkspaceContextService contextService: IWorkspaceContextService,
 		@ILifecycleService lifecycleService: ILifecycleService,
 		@IWorkbenchLayoutService layoutService: IWorkbenchLayoutService,
-		@ICommandService private readonly commandService: ICommandService,
-	) {
+		@ICommandService private readonly commandService: ICommandService) {
```

Comment:
let's not make unnecessary changes under `vs` since these are potential merge conflicts in the future.  Please revert unneeded changes like this and new blank line below.